.travis.yml

```diff
@@ -1,34 +1,13 @@
 language: python
 
-# Run jobs on container-based infrastructure, can be overridden per job
```

Comment:
What's the reason for changing this to the new version (new `jobs` section)?

Reply:
Partly it is the way that all the other SEAMM modules are set up, but primarily the reason is that it breaks up the flow typically into a lint (PEP-8) step, then testing on OS variants, then (for tagged releases) uploading to PyPi, Conda, etc. These steps are ordered and if any step fails subsequent ones are not started, which is a bit nicer to Travis in using their resources.

Travis initially had only the matrix approach, then found it to be not flexible enough, so added the job approach and (I think) are trying to get people to migrate.